.github/workflows/pull-request.yml

```diff
@@ -51,6 +51,12 @@ jobs:
           kind get kubeconfig > /tmp/kubeconfig
           GOPATH=${GITHUB_WORKSPACE}/go make e2e-sanity
 
+      - name: show abnormal events and all objects in konveyor-forklift
+        if: always()
+        run: |
+          kubectl get events --field-selector type!=Normal -A --sort-by='.lastTimestamp'
+          oc get all -n konveyor-forklift -o yaml
```

Comment:
I don't the last part in [the output of job 6721780653](https://github.com/kubev2v/forklift/actions/runs/3931038305/jobs/6721780653)  and we don't assume 'oc' is installed..

Reply:
well the output of the `oc get all` command was really helpful to show all the  pods env variables ie , which is very useful for debugging.  it can be replaced to `kubectl get all`